feature/session/src/main/java/io/github/droidkaigi/confsched2019/session/ui/BottomSheetDaySessionsFragment.kt

```diff
@@ -161,6 +171,16 @@ class BottomSheetDaySessionsFragment : DaggerFragment() {
         }
     }
 
+    override fun onResume() {
+        super.onResume()
+        activity?.addOnBackPressedCallback(onBackPressedListener)
```

Comment:
I think `LifecycleOwner` is not satisfy requirements to other fragments back-key handling.
LifecycleOwner is remove callback instance on destroy in fragment.
But if Overlaid Fragment on this screen, it will be prevented by this screen back-key press handling, because the screen may have focus i guess.

Reply:
Make sense.
Thank you for your detailed explanation 🙇